integrate-kobiton-travisci.md

```diff
@@ -0,0 +1,165 @@
+# Run Kobiton Automation Test on Travis CI
+This guide will demonstrate how to automation test on Kobiton devices with Travis CI.
+
+## Table of contents 
+  - [Prerequisites](#prerequisites)
+  - [1. Configure Travis CI with Github repository](#1-configure-travis-ci-with-github-repository)
+  - [2. Configure automation test script](#2-configure-automation-test-script)
+  - [3. Run automation script on Kobiton devices](#3-run-automation-script-on-kobiton-devices)
+  - [4. Get session information through REST API](#4-get-session-information-through-REST-API)
+  - [5. (Optional) Travis CI customization](#5-optional-travis-ci-customization)
+  - [6. Feedback](#6-feedback)
+
+## Prerequisites
+  - Kobiton account
+
+## 1. Configure Travis CI with Github repository
+
+  This part will guide you on how to sync Travis CI with a Github repository. If you already know how to do it, skip this step.
```

Comment:
Remember to mention that user must have a blank/empty Automation repository before integrate with Kobiton